### ISSUE TITLE
Speed of Webserver degrade after a while

### ISSUE THREAD

**Issue by Jayden9912** [naurril/SUSTechPOINTS#191]
Hi Naurril. May I know did you encounter this situation where the loading of pcd file and images just get super slow after a while (random, it could be one days, could be few hours). This can only be solved by restarting the server.

**Comment by naurril** [comment]
yes a few times and  I don't know why either.  did you notice any strange prints in server log?

**Comment by Jayden9912** [comment]
hi @naurril . I try to disable tools.caching.on in the main.py, so far the issue is gone.
